packages/ui/core-components/src/lib/unsorted/ui/Info.svelte

```diff
@@ -9,27 +9,33 @@
 	import chroma from 'chroma-js';
 	import HoverCard from '../../atoms/hover-card/HoverCard.svelte';
 
-	/** @type {import('@steeze-ui/svelte-icon').IconSource} */
 	let icon = InfoCircled;
-
 	export let description = '';
-
 	export let size = 4;
-
 	export let className = undefined;
-
 	const { resolveColor } = getThemeStores();
 	export let color = 'base-content-muted';
+
 	$: colorStore = resolveColor(color);
 	$: textColor = chroma($colorStore).css();
+
+	// State for manual toggle
+	let isOpen = false;
+
+	function toggleOpen() {
+		isOpen = !isOpen;
+	}
 </script>
 
-<HoverCard align="start" side="right" alignOffset={-8} sideOffset={4}>
+<HoverCard open={isOpen} align="start" side="right" alignOffset={-8} sideOffset={4}>
+	<!-- svelte-ignore a11y-click-events-have-key-events -->
+	<!-- svelte-ignore a11y-no-noninteractive-element-interactions -->
```

Comment:
We shouldn't ignore these accessibility issues, lets resolve them before we merge